packages/terra-application-navigation/src/terra-dev-site/test/application-navigation/HeroAndUser.test.jsx

```diff
@@ -2,7 +2,7 @@
 import React from 'react';
 import ApplicationBase from 'terra-application/lib/application-base';
 import ApplicationNavigation from '../../../ApplicationNavigation';
-import Placeholder from 'terra-doc-template/lib/Placeholder';
+import { Placeholder } from '@cerner/terra-docs';
 
 const hero = <Placeholder title="Test Hero" variant="light" />;
```

Comment:
Should `variant` be removed since it is no longer used in the new placeholder? Although the new placeholder color is a bit hard to read on a dark background.

https://github.com/cerner/terra-framework/pull/1277/files#diff-f1e4767c56d4f935f931188e147e6d7a